docs/registering-ids.md

```diff
@@ -3,38 +3,35 @@ id: registering-ids
 
+The basic set of data that must be registered for each sample consists of:
 * Submitter Donor ID
 * Donor Gender
 * Submitter Specimen ID
+* Specimen Tissue Source
 * Specimen Tumor or Normal Designation
-* Specimen Type
 * Submitter Sample ID
 * Sample Type
 
+During sample registration, **ARGO IDs** will be assigned to your programs data. Any attempts to submit data  which does not refer to a previously registered samples will result in an error, and will prompt you to complete the registration step before proceeding.
 
-To view the registration file schema and  permissible field values, please see the [Data Dictionary](/dictionary). At the point of registration, the  **ARGO IDs** will be generated, and associated to the  your program's submitter IDs.  Any attempts to submit data  which does not refer to a previously registered identifier will result in an error, and will prompt you to complete the registration step before proceeding. Please [contact the DCC](/contact) if you need to make any changes to previously registered data.  
-
-
-## How To
-1. Navigate to the **Data Submissions** area in the top menu.
-2. Click on the **Register Samples** section in the left menu pane for your program.
-3. You can download the **TSV Template** for the registration file and start formatting your file according to the [Data Dictionary](/dictionary) specifications .
-4. Once you have formatted your data file, click **Upload File** and select the file from your browser. Only TSV file types are supported, and the file name must begin with the word *registration*.
-
-5. If there are any errors in your file, they will be displayed upon upload attempt. The error report will also be available for download. Please make sure to download the errors before you navigate away from the page. You must fix all of the errors which are listed, and upload the file again.
+## How To Register Samples
+1. Navigate to the **Submission** area in the top menu.
+1. Click on the **Register Samples** section in the left menu pane for your program.
+1. Download a **TSV Template** of the registration file and formatting the file according to the [Data Dictionary](/dictionary) specifications.
+1. Once your file is formatted, click **Upload File** and select the file from your browser. Only TSV file types are supported, and the file name must begin with the *sample-registration* and end with _.tsv_.
+1. If there are any errors in your file, they will be displayed upon upload attempt. You must fix all of the errors which are listed, and reupload the fixed registration file.
+1. Valid files will be available for preview in the Submission UI. You can review the file contents and filter on new vs. previously registered samples using the star filter.
+1. Once you have reviewed the file, click on the **Register Samples** button to submit your samples.
```

Comment:
...click the "Register Samples" button in the top instructions box to submit you samples.